js/index.js

```diff
@@ -0,0 +1,65 @@
+//Model
+
+var input =
+// changed value to be a property
+    {
+        'value': 0,
+        'aboveOneHundred': false,
+        'belowZero': false
+    }
+  //removed comma
+
+
+//View - needs to reference template and compile - takes list up top and turns it into html
+function renderNumber() {
+$('#Number').html(input.value);
+var source = $('#countly-template').html();
+//removed input on click will render each individually
+var template = Handlebars.compile(source);
+}
```

Comment:
Here, you aren't actually using your template, in order to use it we have to rearrange this a bit:

```js
function renderNumber() {
   var source = $('#countly-template').html();
   var template = Handlebars.compile(source);
   var renderedTemplate = template(input);
 
   $('#Number').html(renderedTemplate);
}